### ISSUE TITLE
https://donate.keikaavousi.com/ donation resource is down

### ISSUE THREAD

**Issue by meta-panic** [keikaavousi/fake-store-api#69]
Hello! I noticed that https://donate.keikaavousi.com/ is down

**Comment by keikaavousi** [comment]
Yes, it has some domain problems. you can temporarily use this wallet to donate until the website is available again.
Thanks in advance!

======Tether(TRC20)======

THer44V7ZdTm35Tco7f5VYCzA4Y3gQPH4u
